tests/Http/Controllers/Api/Geojson/ProjectImagesControllerTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Biigle\Tests\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use ApiTestCase;
+use Faker\Factory as Faker;
+use Biigle\{Label, Image, Annotation, AnnotationLabel};
+use GeoJson\GeoJson as GeoJson;
+use Biigle\Tests\ProjectTest;
+use Biigle\Tests\VolumeTest;
+use Biigle\Tests\LabelTest;
+use Biigle\Tests\ImageTest;
+use Biigle\Tests\AnnotationTest;
+use Biigle\Tests\AnnotationLabelTest;
+
+class ProjectImagesControllerTest extends ApiTestCase
+{
+  public function testIndex()
+  {
+    $faker = Faker::create();
+    $firstProject = $this->project();
+    $secondProject = ProjectTest::create();
+
+    $volume = $this->volume();
+    $volume_2 = VolumeTest::create();
+    $secondProject->volumes()->save($volume);
+
+
+    $label_names = ["sponge", "jellyfish", "starfish"];
```

Comment:
Please use camelCase variable names.

```suggestion
    $labelNames = ["sponge", "jellyfish", "starfish"];
```